javascript/builtins/Intl.json

```diff
@@ -1263,6 +1263,831 @@
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "nodejs": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": false
+                },
+                "opera_android": {
+                  "version_added": false
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": false
+                },
+                "samsunginternet_android": {
+                  "version_added": false
+                },
+                "webview_android": {
+                  "version_added": "74"
+                }
+              }
+            }
+          },
+          "collation": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Locale/collation",
+              "spec_url": "https://tc39.github.io/proposal-intl-locale/#sec-Intl.Locale.prototype.collation",
+              "support": {
+                "chrome": {
+                  "version_added": "74"
+                },
+                "chrome_android": {
+                  "version_added": "74"
+                },
+                "edge": {
+                  "version_added": false
+                },
+                "edge_mobile": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": false
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "ie": {
+                  "version_added": false
+                },
+                "nodejs": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": false
+                },
+                "opera_android": {
+                  "version_added": false
+                },
+                "safari": {
+                  "version_added": false
+                },
+                "safari_ios": {
+                  "version_added": false
+                },
+                "samsunginternet_android": {
+                  "version_added": false
+                },
+                "webview_android": {
+                  "version_added": "74"
+                }
+              },
+              "status": {
+                "deprecated": false,
+                "experimental": false,
+                "standard_track": true
+              }
+            }
+          },
+          "hourCycle": {
+            "__compat": {
+              "mdn_url": "https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Locale/hourCycle",
+              "spec_url": "https://tc39.github.io/proposal-intl-locale/#sec-Intl.Locale.prototype.hourCycle",
+              "status": {
```

Comment:
It seems that starting here, `status` was put above `support`.  Can we reverse these?  (I'll only comment on this one.)

Reply:
Sounds good! I'll make the switch 😄